app/views.py

```diff
@@ -166,11 +167,27 @@ class ContractAdd(TemplateView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         case_numbers = self.request.GET.get('case_numbers') or self.kwargs.get('case_numbers')
+        date_from = self.request.GET.get('case_date_from')
+        date_to = self.request.GET.get('case_date_to')
+        contract_data = []
+        if date_from or date_to:
```

Comment:
es mas facil hacer `date_from and date_to` y con un else agregar el mensaje diciendo que tienen que estar las dos fechas en lugar de hacer un try/catch